tests/e2e/framework/kubernetes.go

```diff
@@ -215,16 +215,42 @@ func (k *KubeInfo) generateRbac(src, dst string) error {
 		glog.Errorf("Cannot read original yaml file %s", src)
 		return err
 	}
-	namespace := []byte(fmt.Sprintf("namespace: %s", k.Namespace))
-	r := regexp.MustCompile("namespace: default")
-	content = r.ReplaceAllLiteral(content, namespace)
+
+	content = replacePattern(k, content, "namespace: default",
+		"namespace: "+k.Namespace)
```

Comment:
Can this be a loop for each service account?

Reply:
Agree, it should be written better.